api/v1alpha1/changetransferpolicy_types.go

```diff
@@ -78,8 +78,22 @@ type CommitBranchState struct {
 }
 
 type CommitShaState struct {
-	Sha        string      `json:"sha,omitempty"`
+	// Sha is the SHA of the commit in the branch
+	Sha string `json:"sha,omitempty"`
+	// CommitTime is the time the commit was made
 	CommitTime metav1.Time `json:"commitTime,omitempty"`
+	// RepoURL is the URL of the repository where the commit is located
+	RepoURL string `json:"repoURL,omitempty"`
+	// Commands are the commands that were run to hydrate the branch
+	Commands []string `json:"commands,omitempty"`
```

Comment:
Maybe drop this? I can't think of a scenario where it's relevant in the promoter context. More of a hydrator thing.

Reply:
Possibly... but the ultimate source of truth is the manifests. If I'm a user, I'm probably going straight to the diff rather than reading a long list of helm CLI flags.

Reply:
I think of the database use case aka the diff would not contain much maybe you have something like

```
air-generate 1.5.3
```

the 1.5.3 comes from a lookup in a database for manifests with that version